### ISSUE TITLE
4.4.1 traceback with python 3.5.2

### ISSUE THREAD

**Issue by battlemidget** [aio-libs/multidict#286]
**Describe the bug**

4.4.1 ends in traceback with python 3.5.x

```
    from multidict import istr
  File "/run/code/.tox/py3/lib/python3.5/site-packages/multidict/__init__.py", line 18, in <module>
    from ._abc import MultiMapping, MutableMultiMapping
  File "/run/code/.tox/py3/lib/python3.5/site-packages/multidict/_abc.py", line 37, in <module>
    class MutableMultiMapping(MultiMapping[_T], MutableMapping[str, _T]):
  File "/run/code/.tox/py3/lib/python3.5/site-packages/multidict/_abc.py", line 44, in MutableMultiMapping
    def extend(self, *args: _Arg[_T], **kwargs: _T) -> None:
  File "/usr/lib/python3.5/typing.py", line 546, in __getitem__
    "Cannot subscript an existing Union. Use Union[u, t] instead.")
TypeError: Cannot subscript an existing Union. Use Union[u, t] instead.
```

**To Reproduce**   

**Expected behavior**

**Logs/tracebacks**

**Your version of the Python**

3.5.2

**Your version of the multidict distribution**

4.4.1

**Additional context**

**Comment by sergeychibunin** [comment]
I have the same bug.

ubuntu 16.04
python 3.5.2

**Comment by asvetlov** [comment]
The problem was fixed by Python 3.5.3
Please upgrade python version or use older multidict instead

**Comment by webknjaz** [comment]
https://github.com/aio-libs/multidict/blob/master/setup.py#L272 let's update this then

**Comment by asvetlov** [comment]
Give me some time.
Maybe we can keep old pythons support

**Comment by asvetlov** [comment]
Fixed by #289
